src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -0,0 +1,175 @@
+    }
+
+    @Test
+    public void sinceDate_correctFormat_success() throws ParseException {
+        String sinceDate = "01/07/2017";
+        String input = DEFAULT_MANDATORY_ARGS + String.format("-since %s", sinceDate);
+        CliArguments cliArguments = ArgsParser.parse(input.split(INPUT_DELIMITER));
+
+        Date expectedSinceDate = TestUtil.getDate(2017, Calendar.JULY, 1);
+        Assert.assertEquals(expectedSinceDate, cliArguments.getSinceDate().get());
+    }
+
+    @Test
+    public void untilDate_correctFormat_success() throws ParseException {
+        String untilDate = "30/11/2017";
+        String input = DEFAULT_MANDATORY_ARGS + String.format("-until %s", untilDate);
+        CliArguments cliArguments = ArgsParser.parse(input.split(INPUT_DELIMITER));
+
+        Date expectedUntilDate = TestUtil.getDate(2017, Calendar.NOVEMBER, 30);
+        Assert.assertEquals(expectedUntilDate, cliArguments.getUntilDate().get());
+    }
+
+    @Test(expected = ParseException.class)
+    public void emptyArgs_throwsParseException() throws ParseException {
+        ArgsParser.parse(new String[]{});
+    }
+
+    @Test(expected = ParseException.class)
+    public void missingMandatoryConfigArg_throwsParseException() throws ParseException {
+        String input = String.format("-output %s -since 01/07/2017 -until 30/11/2017", OUTPUT_DIRECTORY_ABSOLUTE);
+        ArgsParser.parse(input.split(INPUT_DELIMITER));
+    }
+
+    @Test(expected = ParseException.class)
+    public void absoluteDirectory_forConfigFile_throwsParseException() throws ParseException {
+        String absDirectory = Paths.get(PROJECT_DIRECTORY).getParent().toString();
+        String input = String.format("-config %s", absDirectory);
+        ArgsParser.parse(input.split(INPUT_DELIMITER));
+    }
+
+    @Test(expected = ParseException.class)
+    public void relativeDirectory_forConfigFile_throwsParseException() throws ParseException {
+        String relDirectory = Paths.get(PROJECT_DIRECTORY).getParent().toString();
+        String input = String.format("-config %s", relDirectory);
+        ArgsParser.parse(input.split(INPUT_DELIMITER));
+    }
+
+    @Test(expected = ParseException.class)
+    public void parse_absoluteConfigFileNotExists_throwsArgumentParserException() throws ParseException {
+        String absConfigFile = Paths.get(PROJECT_DIRECTORY, "somefile_not_existent.csv").toString();
+        String input = String.format("-config %s", absConfigFile);
+        ArgsParser.parse(input.split(INPUT_DELIMITER));
+    }
+
+    @Test(expected = ParseException.class)
+    public void parse_relativeConfigFileNotExists_throwsArgumentParserException() throws ParseException {
+        String relConfigFile = Paths.get("somefile_not_existent.csv").toString();
+        String input = String.format("-config %s", relConfigFile);
+        ArgsParser.parse(input.split(INPUT_DELIMITER));
```

Comment:
I think absolute is sufficient for this case. How about testing empty config param?
`-config `